examples/python-cicd-with-cloudbuilder/my_module/my_module.py

```diff
@@ -0,0 +1,72 @@
+"""
+   Copyright 2019 Google LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+Sample module used to demonstrate CI with:
+ - Pytest
+ - Cloud Source Repositories
+ - Cloud Builder
+
+Tutorial found in README.md
+
+
+
+"""
+import numpy as np
+
+
+def add(a, b):
+  """
+  Adds two numbers, a and b
+  :param a  First number to add
+  :param b  Second number to add
+  :return: The sum of a and b
+  """
+  return a + b
+
+
+def square(x):
+  """
+  returns the square of x
+  :param x: number to square
+  :return: x ** 2
+  """
+  return x ** 2
+
+
+def log_transform(x, const=1):
+  """
+  returns the log of x to reduce skewedness in our dataset
+  :param x: value or values to transform
+  :param const: a constant to add to x to prevent taking the log of 0
+  :return: log of x + 1
+  """
+  assert const > 0
```

Comment:
See the styleguide for why an exception should be used here instead of assert https://google.github.io/styleguide/pyguide.html#244-decision . Probably worth putting that in the test as well, it would be good to have a nice example assertion test.

Reply:
Make sure you check out the styleguide section on exceptions as well: https://google.github.io/styleguide/pyguide.html#24-exceptions .

Reply:
If you feel it is out of scope for what you're trying to do here, push back, we don't need to turn this into a set of unit test examples  (though we could use that which is why I'm asking for an exception test).

Reply:
I'm good on the code side, I have some questions on the test side, I dropped them into chat.  This one isn't quite done yet.